lib/rpmsg/rpmsg_core.c

```diff
@@ -352,8 +352,10 @@ void rpmsg_ns_callback(struct rpmsg_endpoint *ept, void *data,
 			_ept->destroy_cb(_ept);
 
 		rpmsg_destroy_ept(_ept);
+#if 0
```

Comment:
We need to decide how we want to handle this.  Do we need to keep track of how and ept was created so we can decide how/if to destroy it?